data/formats-data.ts

```diff
@@ -33,6 +33,7 @@ export const FormatsData: {[k: string]: SpeciesFormatsData} = {
 		randomBattleLevel: 82,
 		randomDoubleBattleMoves: ["airslash", "heatwave", "overheat", "protect", "scorchingsands", "tailwind"],
 		randomDoubleBattleLevel: 80,
+		randomBattleNoDynamaxMoves: ["defog", "earthquake", "fireblast", "hurricane", "roost", "toxic"],
```

Comment:
this is getting somewhat ridiculous.... i don't really like containing all random battles elements in formats-data.ts and was aiming on making the file eventually obsolete apart from tiering.

Reply:
I don't have time to review this PR tonight but I have plans to move random battles data (including selection logic for moves/item/abilities/etc) into its own data file and just have one random-teams.ts file. That would solve the issue of clutter in formats-data as well as separation of code and data in randbats (making it easier for inexperienced people to understand the logic and cleaning things up a lot for experienced programmers). I'm unsure of when I'll have time to work on that though, perhaps this summer?